third_party/2and3/dateparser.pyi

```diff
@@ -0,0 +1,13 @@
+import datetime
+from typing import Optional, List, Dict
+
+__version__ = str
+
+def parse(
+    date_string: str,
+    date_formats: Optional[List[str]] = None,
```

Comment:
From the code it looks like these list arguments can also be tuples or sets; there is an isinstance check in `DateDataParser.__init__`.

Reply:
I see this now.  The documentation says the arguments are lists which confused me.  I've tried to make this change in the latest revision (which I've force pushed).